### DISCUSSION TITLE
Allow Captions for WYSIWYG Images

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#14645] [Feature Requests]
From v9.14.3, WYSIWYG image options were removed to [fix edited state](https://github.com/directus/directus/issues/14372) ([PR](https://github.com/directus/directus/pull/14404))

Image options modal allowed user-friendly editing of alt text, enabling captions (that correctly wrap the image in HTML `<figure>` with `<figcaption`), and advanced styling.
<img width="480" alt="Screen Shot 2022-07-25 at 1 16 11 PM" src="https://user-images.githubusercontent.com/77066045/180875231-5b071e24-f471-465b-a274-7fed66a76371.png">

**Comment by rijkvanzanten** [comment]
Cc @Nitwel / @licitdev

**Comment by licitdev** [comment]
@barrylow I've just tried reverting the changes from PR #14404 but I am unable to reproduce the view in your screenshot.

![image](https://user-images.githubusercontent.com/26413686/181044078-715a950e-bca8-44df-8c1c-8e09664ad5fa.png)

Is there any custom configuration / setup involved?

**Reply by licitdev** [reply]
Found out that the `{ "image_caption": true }` tinymce override enables this feature (Ref: #15040).
Currently the image caption is unsupported natively.

**Comment by SimonVillage** [comment]
Can we get this back? It's a bit sad that this feature was existing but got removed just to fix something else?

**Reply by licitdev** [reply]
It's unfortunate, but image captions wasn't officially supported in our custom image interface. 
It would be nice to add support for this feature. 👍🏼 
Kindly refer to https://docs.directus.io/getting-started/support.html#why-haven-t-you-added-this-feature-or-fixed-that-issue-yet.

When this feature gets implemented, it should have a similar behaviour as tinymce's image plugin.
https://github.com/tinymce/tinymce/blob/0a65f9a65466ca27dd0f928192e9df6defd6fa47/modules/tinymce/src/plugins/image/main/ts/core/ImageData.ts#L72-L95

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
